CODEOWNERS

```diff
@@ -0,0 +1,57 @@
+# This file controls automatic PR reviewer assignment. See the following docs:
+#
+# * https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+# * https://docs.github.com/en/organizations/organizing-members-into-teams/managing-code-review-settings-for-your-team
+#
+# The goal of this file is for most PRs to automatically and fairly have one
+# maintainer and two reviewers set as PR reviewers. All maintainers have
+# permission to approve and merge PRs, but reviewers do not. Most PRs should by
+# members of the reviewers group before being passed to a maintainer for final
+# review.
+#
+# This in part depends on how the groups in this file are configured.
+#
+# @crossplane/steering-committee - Assigns 3 members. Admin perms to this repo.
+# @crossplane/crossplane-maintainers - Assigns 1 member. Maintain perms to this repo.
+# @crossplane-crossplane-reviewers - Assigns 2 members. Triage perms to this repo.
```

Comment:
the other too are `crossplane/...` but this one is `crossplane-...` - typo?

Reply:
Aha good catch. I had remembered there was something I wanted to fix in here but not what it was.